README.md

```diff
@@ -59,72 +59,68 @@ repository, and the results are hosted on the
   * [R benchmarks](https://github.com/conbench/conbench#example-r-benchmarks)
 
 
-## Contributing
+## Developer environment
 
+### Dependencies
 
-### Developer environment
+- [`make`](https://www.gnu.org/software/make/), [`docker compose`](https://docs.docker.com/compose/install/): common developer tasks depend on these tools. They need to be set up on your system.
+- `GITHUB_API_TOKEN` environment variable: set up a GitHub API token using [GitHub's instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). It's recommended to only give the token read-only permissions to public repositories (which is the default for fine-grained personal access tokens). Run `export GITHUB_API_TOKEN="token"` in your current shell.
```

Comment:
```suggestion
- `GITHUB_API_TOKEN` environment variable: set up a GitHub API token using [GitHub's instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). It's recommended to only give the token read-only permissions to public repositories (which is the default for fine-grained personal access tokens). Run `export GITHUB_API_TOKEN="{token}"` in your current shell.
```

Makes it slightly more clear that the TOKEN value should be swapped out?

Reply:
:+1: I pressed the "commit suggestion" button :ship: